src/main/java/uk/gov/hmcts/reform/idam/service/aop/MethodRetryAspect.java

```diff
@@ -36,6 +36,7 @@ public Object retry(ProceedingJoinPoint joinPoint) throws Throwable {
                 successful = true;
             } catch (FeignException fe) {
                 if (retryAttempts >= 0 && fe.status() == HttpStatus.SC_UNAUTHORIZED) {
+                    log.info("Security Util generate tokens are going to be called from retry");
```

Comment:
This log looks like debug log that might not necessarily help to identify a potential issue. Please try to log more meaningful information such as - which endpoint has thrown the exception for example.